VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/participant-waiting-room.component.scss

```diff
@@ -57,6 +65,13 @@
 
 .header-content-grid-with-lock {
   display: grid;
-  grid-template-columns: 200px 100px auto auto;
+  grid-template-columns: 100px 100px auto auto;
+
+  @media only screen and(min-width: $breakpoint-small) and (max-width: $breakpoint-medium) {
+    grid-template-columns: 200px 100px auto auto;
+  }
+  @media only screen and (max-width: $breakpoint-medium) {
+    grid-template-columns: 250px 100px auto auto;
+  }
```

Comment:
Should the second one here be min-width: $breakpoint-medium? Currently the first one won't ever be used.
If that's the change that should be made you should also remove the max width rule from the first.

You could also look at an alternative using fr. And not use the media queries at all, or just use them to set a total width.

grid-template-columns: 1fr 100px auto auto;

Reply:
And actually change the px values to rem. I know I mostly focused on updating the font size but there's a few places I could see pixels causing issues if someone has a different font size and this could be one. It would actually be worth using for everything except some borders.